content/guides/guides/command-line.md

```diff
@@ -657,6 +657,19 @@ cypress verify
 ✔  Verified Cypress! /Users/jane/Library/Caches/Cypress/3.0.0/Cypress.app
 ```
 
+To change the default timeout of 30 seconds, you can set the environment
+variable `CYPRESS_VERIFY_TIMEOUT`:
+
+```shell
+export CYPRESS_VERIFY_TIMEOUT=60000 # wait for 60 seconds
+cypress verify
+```
+
+Note that the `cypress verify` command is executed as part of the `cypress open`
```

Comment:
Is this always ran on these commands or only in certain scenarios? I might be helpful to add this note in reverse under the `cypress open` & `cypress run` sections to add a link to this section

Reply:
@tbiethman I think it's only run in certain conditions - like if we can't immediately find it installed? Would have to look at code.

Reply:
The timeout only comes into play when the verify smoke test executes. We run the smoke test when:
* We run the `verify` command directly with --dev option
  * this bypasses the binary check and forces the smoke test to run
* We run the `run`/`open`/`verify` commands and the binary has not been previously validated
  * if not previously validated, the smoke test is run as part of the binary validation process. 
  * there is logic for a --force option in here to force the validation, but it's no longer supported and errors if provided.